resolwe_bio/tools/rnaseq.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
```

Comment:
User Python 3 wherever possible or we'll have a lot of problems in the near future.